### ISSUE TITLE
Fields in Baloon.Builder incorrectly marked as @Dp but store actually pixels.

### ISSUE THREAD

**Issue by dekalo-stanislav** [skydoves/Balloon#79]
**Please complete the following information:**
- Library Version [1.1.9]
- Affected Device(s) [Pixle 2]
 
**Describe the Bug:**
Form Baloon.Builder
```
@JvmField @Dp
var arrowSize: Int = context.dp2Px(12)
```
Above it is clearly visible that arrowSize contains 12dp converted to px by default.
and 
```
fun setArrowSize(@Dp value: Int): Builder = apply { this.arrowSize = context.dp2Px(value) }
```
`@Dp value: Int` - converted to Px and stored to `@Dp var arrowSize: Int` also marked as `@Dp`.

From above it is clearly visible that `arrowSize` field is incorrectly marked as `@Dp`.

This mostly affects kotlin DSL example:
```
createBalloon(activity) {
            ... 
            arrowSize = 4 // it will set actually 4px not 4dp
            ...
        }.showAlignBottom(anchor)
```

**Expected Behavior:**
If field marked as `@Dp` it should store `DP`.

**Proposal**
1. Mark each incorrect `@Dp` field as `@Px`
2. Reimplement logic so it will actually store DP and make DP -> PX conversion on the implementation side
3. Add backing fields https://kotlinlang.org/docs/reference/properties.html#backing-fields
something like this
```
    @Px
    internal var _arrowSize: Int = 2

    var arrowSize: Int
        @Dp get() = context.px2Dp(_arrowSize)
        @Dp set(value) {
            _arrowSize = context.dp2Px(value)
        }
```

**Comment by skydoves** [comment]
Hi, @dekalo-stanislav !
I'm aware of the issue, but I think the property must keep the `@JvmField` annotation for supporting java.
I think the `@Dp` annotations should be `@Px` in the next release. 
If you have any other suggestions, let me know your thinkings!

**Comment by skydoves** [comment]
Hi, @dekalo-stanislav!
This issue is fixed in the new version [1.2.0](https://github.com/skydoves/Balloon/releases/tag/1.2.0).
Thank you for your issue and suggestion. :)
